components/buttons.tsx

```diff
@@ -0,0 +1,42 @@
+import type { FC } from "react";
+import React, { type ReactElement } from "react";
```

Comment:
```suggestion
import type { FC, ReactElement } from "react";
```
...and don't import all of React

Reply:
Has been fixed